pom.xml

```diff
@@ -2672,23 +2676,32 @@
     </profile>
     <!-- Profiles for specific JDK versions -->
     <profile>
-      <id>java8</id>
+      <id>java9-or-later</id>
       <activation>
-        <jdk>[,8]</jdk>
+        <jdk>[9,]</jdk>
       </activation>
       <properties>
-        <maven.compiler.source>${javac.version}</maven.compiler.source>
-        <maven.compiler.target>${javac.version}</maven.compiler.target>
+        <!-- supported since Java 9 -->
+        <maven.compiler.release>${javac.version}</maven.compiler.release>
```

Comment:
Also set in global `properties`, so I guess we can remove the `java9-or-later` profile.